docs/source/markdown/links/podman-play.1

```diff
@@ -0,0 +1 @@
+.so man1/podman-kube.1
```

Comment:
I don't think think `podman play` should point to `podman kube`.  New commands of `kube` won't be added to `play` so they will diverge at some point.

Should we just alias to `podman-kube-play`?

@rhatdan WDYT?